README.md

```diff
@@ -14,10 +14,10 @@ Then add the following line to your .bashrc file:
 
 That will create some directories inside your $HOME/.evm dir:
 
-- **erlang_tars** : the place where all erlang tarballs downloaded by evm will be cached.
+- **erlang_tars** : the place where all the erlang tarballs downloaded by evm will be cached.
```

Comment:
Thanks for the help, but you need to be consistent. Here you are adding the 'the' while on line 44 you are removing.
English is clearly not my main language, but I guess both situations are grammatically equivalent, aren't they?